templates/search/blocks/search_filters.html

```diff
@@ -111,6 +114,7 @@ <h4 class="tna-heading search-filters__accordion-section-heading" id="record_ope
                         </legend>
 
                         <div class="search-filters__opening-date-section">
+                            <fieldset>
                             <span class="example-text">For example, 27 3 2007</span><br>
```

Comment:
I think each of these need a unique `id` so we can add `aria-describedby="whatever-id"` to the `<fieldset>`

Reply:
id's have now been applied